### DISCUSSION TITLE
It appears that https://open-props.style/ is down

### DISCUSSION THREAD

**Discussion by derekjwilliams** [argyleink/open-props#497] [General]
I've been getting a firebase error page all day today when navigating to https://open-props.style/

Firebase error page displayed:

![Screenshot 2024-04-30 at 14 15 06](https://github.com/argyleink/open-props/assets/91456/5c220515-0343-4289-acad-54a97b8927ff)

**Comment by argyleink** [comment]
it's back! thank you, CI/CD back in action 👍🏻
